components/security-mgt/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/internal/SecurityMgtServiceComponent.java

```diff
@@ -62,24 +63,17 @@ public static ConfigurationContext getServerConfigurationContext() {
     @Activate
     protected void activate(ComponentContext ctxt) {
         try {
-            ConfigurationContext mainConfigCtx = configContextService.getServerConfigContext();
-            AxisConfiguration mainAxisConfig = mainConfigCtx.getAxisConfiguration();
             BundleContext bundleCtx = ctxt.getBundleContext();
-            String enablePoxSecurity = ServerConfiguration.getInstance()
-                    .getFirstProperty("EnablePoxSecurity");
-            if (enablePoxSecurity == null || "true".equals(enablePoxSecurity)) {
-                mainAxisConfig.engageModule(POX_SECURITY_MODULE);
-            } else {
-                log.info("POX Security Disabled");
-            }
-
-            bundleCtx.registerService(SecurityConfigAdmin.class.getName(),
-                    new SecurityConfigAdmin(mainAxisConfig, registryService.getConfigSystemRegistry(), null), null);
-            bundleCtx.registerService(Axis2ConfigurationContextObserver.class.getName(),
-                    new SecurityAxis2ConfigurationContextObserver(), null);
-            bundleCtx.registerService(STSAdminServiceInterface.class, new STSAdminServiceImpl(), null);
             bundleCtx.registerService(KeyStoreManagementService.class.getName(), new KeyStoreManagementServiceImpl(),
                     null);
+            try {
+                addKeystores();
+            } catch (Exception e) {
+                String msg = "Cannot add keystores";
```

Comment:
```suggestion
                String msg = "Error while adding key stores.";
```
better to add some more info as well.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/2921/commits/7b2708081125a410e0d660cf4f065fd3131049fd.